plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -225,8 +225,7 @@ func defaultPriorities() sets.String {
 		// Prioritizes nodes that have labels matching NodeAffinity
 		factory.RegisterPriorityFunction2("NodeAffinityPriority", priorities.CalculateNodeAffinityPriorityMap, priorities.CalculateNodeAffinityPriorityReduce, 1),
 
-		// Taints are restrictive markers added to Nodes. Only Pods which explicitly tolerate a particular taint
-		// will be scheduled onto a Node marked with that taint.
+		// Prioritizes nodes that marked with particular taint explicitly tolerated by Pods.
```

Comment:
“Prioritizes nodes that marked with taint which pod can tolerate. ” That may be fine.
BTW, squah the commit.

Reply:
OK. Thanks.